pyogrio/arrow_bridge.h

```diff
@@ -0,0 +1,109 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Would probably be good to add a comment with the date this was copied from the source URL - though I guess if it was copied instead from a version modified in GDAL, good to provide a direct link to that and date

Reply:
There was already a comment about that (a bit below), but I now updated this file to match the latest version of upstream arrow (with the inclusion guards) and added the commit and date of that version